examples/streaming/kafka/src/main/scala/org/apache/gearpump/streaming/examples/kafka/wordcount/KafkaWordCount.scala

```diff
@@ -46,7 +47,10 @@ object KafkaWordCount extends App with ArgumentsParser {
     val sumNum = config.getInt("sum")
     val kafkaStreamProcessorNum = config.getInt("kafka_stream_processor")
 
-    val kafkaConfig = KafkaConfig(ConfigFactory.parseResources("kafka.conf"))
+    val kafkaConfig = new KafkaConfig(
```

Comment:
confused. What is KafkaConfig? kafka.conf or the combination of kafka.conf, consumer.properties and producer.properties?